open_dread_rando/dread_patcher.py

```diff
@@ -90,7 +90,8 @@ def chunks(l, n):
         "linear_dps": configuration.get("linear_dps"),
         "configuration_identifier": lua_util.wrap_string(configuration_identifier),
         "required_artifacts": configuration["objective"]["required_artifacts"],
-        "enable_death_counter": cosmetic_options["enable_death_counter"]
+        "enable_death_counter": cosmetic_options["enable_death_counter"],
+        "enable_room_ids": cosmetic_options["enable_room_ids"]
```

Comment:
I like to keep a comma on the last line, removes the need to edit the line above when adding a new entry.